### ISSUE TITLE
LLLLVAR missing

### ISSUE THREAD

**Issue by RZProf** [CBidis/CSharp8583#4]
in parsing some iso8583 data I faced field number 48 has LengthType.LLLLVAR and unfortunately not supported  by library. is it possible to handle values with current version?!

message class looks like below:

```
public class IsoMessage : BaseMessage
    {
           .
           .
           .
           [IsoField(IsoFields.F48, 9999, LengthType.LLLLVAR, ContentType.ANS)]
           public virtual string AdditionalDataPrivate { get; set; }
           .
           .
           .
    }
```

**Comment by RZProf** [comment]
when i check source code. there is LLLLVAR in LengthType enum but in nuget package is not implemented

**Comment by CBidis** [comment]
Good evening @RZProf we have released a major bump upgrade 2.0.0 that contains all commits from contributors plus support for NET.6 and higher versions 🙏
